darwin-op/motion-manager.md

```diff
@@ -1,24 +1,24 @@
 ## Motion Manager
 
-This manager implement the `DARwInOPMotionManager` class and allows you to play
+The `DARwInOPMotionManager` class allows you to play
 a predefined motion stored in the "motion\_4096.bin" file. The main motions and
 their corresponding ids are listed in [appendix](motions-files.md).
 
 It is also possible to add custom motions to this file by using the `Action
 Editor` tool.
 
-> **note**:
-More informations about this tool provided by ROBOTIS is available on Robotis
+> **Note**:
```

Comment:
Another small note, I agree with you it will be better with an uppercase but we use everywhere in the documentation a lowercase for the note ( we should therefore change everywhere or nowhere to keep consistency)